parquet-protobuf/src/test/resources/BinaryTree.par

```diff
@@ -0,0 +1,50 @@
+message Trees.BinaryTree {
+  optional group value = 1 {
```

Comment:
Aren't groups [deprecated](https://developers.google.com/protocol-buffers/docs/proto#groups)?

Reply:
or is par not proto?

Reply:
this is parquet schema, not proto.  protos should/would have a .proto suffix.